trackers/javascript-tracker/test/integration/youtube.test.ts

```diff
@@ -137,24 +135,25 @@ describe('YouTube Tracker', () => {
     await browser.url('/index.html');
     await browser.setCookies({ name: 'container', value: docker.url });
     await browser.url('/youtube/tracking.html');
-    await browser.waitUntil(() => $('#youtube').isExisting(), {
+    await waitUntil(browser, () => $('#youtube').isExisting(), {
       timeout: 5000,
       timeoutMsg: 'expected youtube after 5s',
     });
 
     const player = $('#youtube');
     await player.click(); // emits 'playbackqualitychange' and 'play';
-    await player.keys(Array(3).fill('ArrowRight')); // Skips to the point just before 'percentprogress' fires
+    await player.keys(Array(2).fill('ArrowRight')); // Skips to the point just before 'percentprogress' fires
```

Comment:
It's interesting that this needed to change, though I'm sure I did something similar in the past to get tests working after they seemingly started failing out of nowhere.